### PR TITLE
New recipe Peasoup

### PR DESCRIPTION
**Pull request by julb562**
Added a new recipe for the very Finnish peasoup

### CODE REVIEW

**Comment by bast** [comment]
One thing you could do is to cross-reference the issue that you created. Is there an issue this PR is trying to solve?

File: soups/Peasoup.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Pea soup
+## Ingredients
```
Comment (gregordecristoforo) [inline soups/Peasoup.md:2]: Nice and simple! Maybe specifying the amount would be helpful.

**Review by gregordecristoforo — APPROVED** [review]
Thanks for the recipe, I just added one suggestion but approve it anyway.

**Comment by cappecaps** [comment]
I think your request and mine can be merged for some reason

**Pull request merged**